### PR TITLE
feat(pci.storage.databases): integrations - change integrations display

### PR DESCRIPTION
**Pull request by jperchoc**
DATATR-31

Signed-off-by: Jonathan Perchoc <jonathan.perchoc@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/datatr-34-integrations-controls`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #DATATR-31
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Update display of integrations with new type

## Related

```
DATATR-30 <- DATATR-34 <- DATATR-31
```
[DATATR-30](https://github.com/ovh/manager/pull/7186)
[DATATR-34](https://github.com/ovh/manager/pull/7185)
[DATATR-31](https://github.com/ovh/manager/pull/7187)

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/databases/components/dashboard-integrations/dashboard-integrations.controller.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+export default class {
+  $onInit() {
+    this.integrationTypes = this.integrations
+      .reduce((integrationsByType, integration) => {
+        const integrationType = integrationsByType.find(
+          (i) => i.type === integration.type,
+        );
```
Comment (vks-ovh) [inline packages/manager/modules/pci/src/projects/project/storages/databases/components/dashboard-integrations/dashboard-integrations.controller.js:7]: const integrationType = integrationsByType.find(
          ({type}) => type === integration.type,
        );

Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/storages/databases/components/dashboard-integrations/dashboard-integrations.controller.js:7]: Changed here: https://github.com/ovh/manager/pull/7187/commits/a079d442f144d18bbbe531ae80e13319d7ed3a05

File: packages/manager/modules/pci/src/projects/project/storages/databases/components/dashboard-integrations/dashboard-integrations.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                    data-translate-values="{type: integration.type}"
+                >
+                </span>
+                <span
+                    class="oui-badge oui-badge_success"
+                    data-ng-bind="integration.count"
+                ></span>
+            </div>
+            <span
+                ng-if="$ctrl.integrationTypes.length === 0"
```
Comment (Dimfacion) [inline packages/manager/modules/pci/src/projects/project/storages/databases/components/dashboard-integrations/dashboard-integrations.html:21]:
```suggestion
                data-ng-if="$ctrl.integrationTypes.length === 0"
```

File: packages/manager/modules/pci/src/projects/project/storages/databases/components/dashboard-integrations/dashboard-integrations.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<oui-tile
+    class="mb-3"
+    data-heading="{{:: 'pci_databases_dashboard_integrations_title' | translate }}"
+>
+    <oui-tile-definition>
+        <oui-tile-description>
+            <div
+                ng-repeat="integration in $ctrl.integrationTypes track by integration.type"
```
Comment (Dimfacion) [inline packages/manager/modules/pci/src/projects/project/storages/databases/components/dashboard-integrations/dashboard-integrations.html:8]:
```suggestion
                data-ng-repeat="integration in $ctrl.integrationTypes track by integration.type"
```

**Review by Dimfacion — APPROVED** [review]

**Review by vks-ovh — APPROVED** [review]

**Pull request merged**